### PR TITLE
Fix: Ray-detection-update

### PR DESCRIPTION
**Pull request by wattsit**
Rays are now directed by player velocity rather then player input. 

Fixes some little issues with wall jumping. 

May need further adjustment for _perfection._

### CODE REVIEW

File: Moonshot/player/States/wall.gd [hunk 1]
```diff
@@ -67,3 +66,4 @@ func jump() -> void:
 		wall_jump = true
 	}
 	_state_machine.transition_to("Move/Air", msg)
+
```
Comment (wattsit) [inline Moonshot/player/States/wall.gd:69]: 69 hehe

Comment (whitew1994WW) [reply Moonshot/player/States/wall.gd:69]: -.-

**Comment by kepler471** [comment]
# Wall jump Y strength increased to 900 here to make ascending a wall feel much nicer

**Review by whitew1994WW — APPROVED** [review]
Yah looks good

**Review by kepler471 — APPROVED** [review]

**Pull request merged**